### PR TITLE
Adds CoP 5-2 CS for Torino-Samarino, Potete, and Leigon-Moigon.

### PR DESCRIPTION
**Pull request by kaincenteno**
**_I affirm:_**
- [x] that I agree to Topaz Next's [Limited Contributor License Agreement](https://github.com/topaz-next/topaz/blob/release/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have read the [Contributing Guide](https://github.com/topaz-next/topaz/blob/release/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/topaz-next/topaz/blob/release/CODE_OF_CONDUCT.md)
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

Source:
https://youtu.be/gVWzFDHf5v8?t=9227 (Thank you Wiggo)

### CODE REVIEW

File: scripts/zones/Beaucedine_Glacier/npcs/Leigon-Moigon.lua [hunk 1]
```diff
@@ -13,7 +13,10 @@ end
 
 entity.onTrigger = function(player, npc)
     local FoiledAGolem = player:getQuestStatus(tpz.quest.log_id.WINDURST, tpz.quest.id.windurst.CURSES_FOILED_A_GOLEM)
+    local copMission = player:getCurrentMission(COP)
```
Comment (wrenffxi) [inline scripts/zones/Beaucedine_Glacier/npcs/Leigon-Moigon.lua:17]:
```
require("scripts/globals/missions")
```

File: scripts/zones/Beaucedine_Glacier/npcs/Potete.lua [hunk 2]
```diff
@@ -14,7 +14,10 @@ end
 
 entity.onTrigger = function(player, npc)
     local FoiledAGolem = player:getQuestStatus(tpz.quest.log_id.WINDURST, tpz.quest.id.windurst.CURSES_FOILED_A_GOLEM)
+    local copMission = player:getCurrentMission(COP)
```
Comment (wrenffxi) [inline scripts/zones/Beaucedine_Glacier/npcs/Potete.lua:18]:
```
require("scripts/globals/missions")
```

File: scripts/zones/Beaucedine_Glacier/npcs/Torino-Samarino.lua [hunk 3]
```diff
@@ -17,11 +17,12 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-
     local FoiledAGolem = player:getQuestStatus(tpz.quest.log_id.WINDURST, tpz.quest.id.windurst.CURSES_FOILED_A_GOLEM)
     local tuningOutProgress = player:getCharVar("TuningOut_Progress")
+    local copMission = player:getCurrentMission(COP)
```
Comment (wrenffxi) [inline scripts/zones/Beaucedine_Glacier/npcs/Torino-Samarino.lua:23]:
```
require("scripts/globals/missions")
```

**Review by wrenffxi — CHANGES_REQUESTED** [review]
Any scripts that reference missions log tables (such as COP) or mission id tables (such as tpz.mission.id.WHATEVER) should be requiring scripts/globals/missions, which defines those tables.

File: scripts/zones/Beaucedine_Glacier/npcs/Torino-Samarino.lua (hunk 3 above)
Comment (kaincenteno) [reply scripts/zones/Beaucedine_Glacier/npcs/Torino-Samarino.lua:23]: thanks for feedback, it worked without it so i forgot to add em ^^;; thanks Wren

**Review by wrenffxi — APPROVED** [review]

**Pull request merged**